### PR TITLE
Issue 31 - Use collective.js.bootstrap

### LINKED ISSUE #31: Depend on collective.js.bootstrap

### PR DESCRIPTION
**Pull request by rodfersou**
closes #31

### CODE REVIEW

File: src/collective/upload/profiles/uninstall/jsregistry.xml [hunk 1]
```diff
@@ -9,7 +9,7 @@
   <javascript remove="True" id="++resource++collective.upload/jquery.fileupload.js"/>
   <javascript remove="True" id="++resource++collective.upload/jquery.fileupload-fp.js"/>
   <javascript remove="True" id="++resource++collective.upload/jquery.fileupload-ui.js"/>
-  <javascript remove="True" id="++resource++collective.upload/vendor/bootstrap.min.js"/>
+  <javascript remove="True" id="++resource++collective.js.bootstrap/js/bootstrap.min.js"/>
```
Comment (hvelarde) [inline src/collective/upload/profiles/uninstall/jsregistry.xml]: @rodfersou you must not remove this here as other add-ons may still be using it.

File: src/collective/upload/profiles/uninstall/cssregistry.xml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0"?>
 <object name="portal_css" meta_type="Stylesheets Registry">
-  <stylesheet remove="True" id="++resource++collective.upload/css/bootstrap.min.css"/>
+  <stylesheet remove="True" id="++resource++collective.js.bootstrap/css/bootstrap.min.css"/>
```
Comment (hvelarde) [inline src/collective/upload/profiles/uninstall/cssregistry.xml]: @rodfersou you must not remove this here as other add-ons may still be using it.

File: src/collective/upload/profiles/default/cssregistry.xml [hunk 3]
```diff
@@ -1,9 +1,9 @@
 <?xml version="1.0"?>
 <object name="portal_css" meta_type="Stylesheets Registry">
-  <stylesheet title="" cacheable="True" compression="safe" cookable="True"
-    enabled="1" expression="" id="++resource++collective.upload/css/bootstrap.min.css" media="screen"
-    rel="stylesheet" rendering="import"/>
+  <stylesheet title="" cacheable="True" compression="none" cookable="True"
```
Comment (hvelarde) [inline src/collective/upload/profiles/default/cssregistry.xml]: @rodfersou installing this resource is a job of collective.js.bootstrap; check collective.cover for instance.

File: src/collective/upload/profiles/default/jsregistry.xml [hunk 4]
```diff
@@ -54,7 +54,7 @@
       expression="" inline="False" authenticated="True" />
 
   <!-- Bootstrap JS -->
-  <javascript id="++resource++collective.upload/vendor/bootstrap.min.js"
+  <javascript id="++resource++collective.js.bootstrap/js/bootstrap.min.js"
```
Comment (hvelarde) [inline src/collective/upload/profiles/default/jsregistry.xml]: @rodfersou installing this resource is a job of collective.js.bootstrap; check collective.cover for instance.

File: src/collective/upload/upgrades/v3/__init__.py [hunk 5]
```diff
@@ -1,6 +1,36 @@
...
+    'old': '++resource++collective.upload/css/bootstrap.min.css',
+    'new': '++resource++collective.js.bootstrap/css/bootstrap.min.css'
+}
+FIX_JS = {
+    'old': '++resource++collective.upload/vendor/bootstrap.min.js',
+    'new': '++resource++collective.js.bootstrap/js/bootstrap.min.js'
+}
+
+
+def fix_resources_references(setup_tool):
```
Comment (hvelarde) [inline src/collective/upload/upgrades/v3/__init__.py]: @rodfersou you should instead remove bootstrap resources and guarantee collective.js.bootstrap is installed, if needed.

**Comment by hvelarde** [comment]
@tcurvelo do you mind taking a look on this? I want to make a new release tomorrow.

**Comment by tcurvelo** [comment]
I don't get it. Is this an issue yet?

**Comment by hvelarde** [comment]
@tcurvelo we're upgrading the package to depend on the other instead of having its own duplicated copy of Bootstrap.

**Pull request merged**